Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.h

```diff
@@ -14,6 +14,62 @@ namespace Gfx {
 
 struct JBIG2LoadingContext;
 
+// E.3 Arithmetic decoding procedure, but with the changes described in
+// Annex G Arithmetic decoding procedure (software conventions).
+// Exposed for testing.
```

Comment:
Can we, at least, put it in a `JBIG2` namespace?

Reply:
Oh, yeah, that's better.